input/pagecontent/mdi-content-transitions.md

```diff
@@ -0,0 +1,27 @@
+# MDI to EDRS
+
+| Name of Profile | Group | change  |  Current Profile   | New Profile  | Comment |
+| :-------------: | ----- | ------- | ------------------ | ------------ | :-----: | 
+|Cause of Death Part 1 |  MDI Profiles  |  -  | [ObservationCauseOfDeathPart1] | [ObservationCauseOfDeathPart1VitalRecordsNew] |   -   |
+|Cause of Death Part 2 |   MDI Profiles  |    -    | [ObservationContributingCauseOfDeathPart2] | [ObservationCauseOfDeathPart2VitalRecordsNew] | - |
+|Death Date |  MDI Profiles  |    -    | [ObservationDeathDate] |  [ObservationDeathDateVitalRecordsNew]  |   -      |
+|How Death Injury Occurred | MDI Profiles |  -  | [ObservationHowDeathInjuryOccurred] | [ObservationInjuryIncidentVitalRecordsNew]  | - |
+|Manner of Death |  MDI Profiles  |    -    | [ObservationMannerOfDeath] | [ObservationMannerOfDeathVitalRecordsNew] |   -      |
+|Decedent Pregnancy |  MDI Profiles  |    -    | [ObservationDecedentPregnancy] | [ObservationDecedentPregnancyVitalRecordsNew] | - |
+|Tobacco Use Contributed to Death |  MDI Profiles  | - | [ObservationTobaccoUseContributedToDeath] |    [ObservationTobaccoUseContributedToDeathVitalRecordsNew] | - |
+|Autopsy Performed Indicator | MDI Profiles  |    -    | [ObservationAutopsyPerformedIndicator] |    -    |   -   |
```

Comment:
Add autopsy location with reference to USCoreLocation.  We will probably add a location profile for this to make it easier to query, just like we did for death location, injury location, and disposition location.